core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorSplit.java

```diff
@@ -40,7 +42,17 @@ default List<HostAddress> getAddresses()
         return List.of();
     }
 
-    Object getInfo();
+    @JsonIgnore
+    default Map<String, String> getMetadata()
```

Comment:
Let's not use `Metadata`. It is too much overloaded term. 
Let's stay with Info. If you do not want to kill SPI method mid release which I think is okish - lets use the `getSplitInfo` name.